### PR TITLE
beginRequest and endRequest when aborted from same thread

### PR DESCRIPTION
**Pull request by njr-11**
Coverage for begin/endRequest on the path where Connection.abort is invoked from the same thread that is using the connection.  This can be within an LTC or a global transaction, and could be on a sharable or unsharable connection.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.jdbc_fat_v43/test-applications/app43/src/jdbc/fat/v43/web/JDBC43TestServlet.java [hunk 1]
```diff
@@ -445,6 +457,131 @@ public void testMultipleUnsharedConnectionsInTransaction() throws Exception {
         }
     }
 
+    /**
+     * Abort an unsharable connection from the same thread that is using it.
```
Comment (alexsm82) [inline dev/com.ibm.ws.jdbc_fat_v43/test-applications/app43/src/jdbc/fat/v43/web/JDBC43TestServlet.java]: This should be shareable connection.

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
Build has 2 test failures which are pre-existing issues in unrelated componentry

**Pull request merged**